src/app/shared/file-download-link/file-download-link.component.html

```diff
@@ -1,4 +1,5 @@
-<a [routerLink]="(bitstreamPath$| async)?.routerLink" class="dont-break-out"
+<ds-access-status-badge [object]="bitstream"></ds-access-status-badge>
```

Comment:
Could you include an input to show/hide the badge on the link?
This component is used in some contexts where it's not appropriate, e.g. [on the Process detail page](https://github.com/DSpace/dspace-angular/blob/a18e03d75414081edbd661ad8d6494ec59d4a1e0/src/app/process-page/detail/process-detail.component.html#L32-L35)